app/models/customer.rb

```diff
@@ -4,6 +4,9 @@ class Customer < ApplicationRecord
 
   has_many :invoices
   has_many :transactions, through: :invoices
+  has_many :merchants, through: :invoices
```

Comment:
After meeting with Dani, do we want to take these out unless we do end up needing them?

Reply:
I feel that I can explain why we will need these relationships in our model